### PR TITLE
DDCE-1610 : PBIK: Improvements to Employee Exclusion Cancellation Con…

### PR DESCRIPTION
**Pull request by kannanekambaram**
…firmation page

# DDCE-1610

Please include a summary / description of the change and which issue it fixes.  Include any relevant user needs, context or links to other PRs related to this PR (eg. acceptance tests, environment config).

## Checklist PR Raiser
- [ ]  I've updated the README if there are functionality changes
- [x]  I've executed the project script to check Dependencies and update them if they're non-breaking changes
- [x]  I've executed Scalastyle and fixed any warnings that can be resolved quickly
- [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
- [x]  I've included appropriate tests with any code I've added (Unit, Integration), and it meets Coverage
- [x]  I've squashed my commits - including the JIRA issue number in the commit message
- [ ]  I've contacted a BA to create a task if any of these actions would result in a larger piece of work

### CODE REVIEW

File: conf/messages.cy [hunk 1]
```diff
@@ -189,7 +189,20 @@ ExclusionImportant.ExcludeEmployee = Ar gyfer cyflogeion sydd wedi’u heithrio,
 ExclusionImportant.Reminder = Drwy gadarnhau, byddwch yn eithrio’r cyflogai hwn rhag cael ei drethu ar {0} drwy’ch cyflogres o 6 Ebrill {1} ymlaen.
 ExclusionRemovalConfirmation.title = Gwiriwch fanylion y cyflogai rydych am ei ychwanegu yn ôl i’r gyflogres
 ExclusionRemovalConfirmation.inset.text = Drwy gadarnhau, byddwch yn dechrau trethu’r cyflogai hwn ar ''{0}'' drwy’ch cyflogres o 6 Ebrill {1} ymlaen.
-ExclusionRemoval.WhatNext.lede = Rydych wedi rhoi gwybod i ni y bydd {0} yn cael {1} wedi’i drethu/threthu drwy eich cyflogres o 6 Ebrill {2} ymlaen.
+#ExclusionRemoval.WhatNext.lede = Rydych wedi rhoi gwybod i ni y bydd {0} yn cael {1} wedi’i drethu/threthu drwy eich cyflogres o 6 Ebrill {2} ymlaen.
```
Comment (yangwong9664) [inline conf/messages.cy:192]: The old content can be removed?

File: conf/messages [hunk 2]
```diff
@@ -199,9 +199,22 @@ ExclusionImportant.ExcludeEmployee = For excluded employees, you need to report
 ExclusionImportant.Reminder = By confirming, you will exclude this employee from having {0} taxed through your payroll from 6 April {1}.
 ExclusionRemovalConfirmation.title = Check the details of the employee you want to add back to payroll
 ExclusionRemovalConfirmation.inset.text = By confirming, you will start taxing this employee for {0} through your payroll from 6 April {1}.
-ExclusionRemoval.WhatNext.lede = You’ve told us that {0} will be having {1} taxed through your payroll from 6 April {2}.
+# ExclusionRemoval.WhatNext.lede = You’ve told us that {0} will be having {1} taxed through your payroll from 6 April {2}.
```
Comment (yangwong9664) [inline conf/messages:202]: The old content can be removed?

File: app/views/exclusion/WhatNextRescind.scala.html [hunk 3]
```diff
@@ -32,49 +32,47 @@
...
-            <li>
-                @Html(messages("ExclusionRescind.WhatNext.p1"))
-            </li>
-
-            <li>
-                @Html(messages("ExclusionRescind.WhatNext.p2"))
-            </li>
+    <div class="govuk-form-group">
+        <p class="govuk-body" id="confirmation-p">
+            @messages("ExclusionRemoval.WhatNext.p1", name, messages("BenefitInKind.label." + uriInformation.iabdValueURLDeMapper(iabdType)), taxYearRange.cy+"" )
```
Comment (yangwong9664) [inline app/views/exclusion/WhatNextRescind.scala.html:48]: What is the use of +""

File: app/views/exclusion/WhatNextRescind.scala.html [hunk 4]
```diff
@@ -32,49 +32,47 @@
...
+        </p>
+        <h2 class="govuk-heading-m">
+            @messages("ExclusionRemoval.whatYouNeedToKnow.subHeading")
+        </h2>
+        <p class="govuk-body">@messages("ExclusionRemoval.whatYouNeedToKnow.p1", taxYearRange.cy+"")</p>
```
Comment (yangwong9664) [inline app/views/exclusion/WhatNextRescind.scala.html:53]: what is the use of +""

**Review by yangwong9664 — CHANGES_REQUESTED** [review]

File: conf/messages.cy (hunk 1 above)
Comment (kannanekambaram) [reply conf/messages.cy:192]: Removed the old content

File: conf/messages (hunk 2 above)
Comment (kannanekambaram) [reply conf/messages:202]: Removed the old content

File: app/views/exclusion/WhatNextRescind.scala.html (hunk 4 above)
Comment (kannanekambaram) [reply app/views/exclusion/WhatNextRescind.scala.html]: removed it and re-run the testing looks fine.

**Review by yangwong9664 — APPROVED** [review]

**Pull request merged**